internal/pkg/daemon/metrics/metrics.go

```diff
@@ -153,7 +157,10 @@ func New() *Metrics {
 				metricsLabelPod,
 				metricsLabelContainer,
 				metricsLabelExecutable,
-				metricsLabelSyscall,
+				metricsLabelPerms,
+				metricsLabelScontext,
+				metricsLabelTcontext,
+				metricsLabelTclass,
```

Comment:
wouldn't doing this dramatically increase the cardinality of this metric? I really think we shouldn't do this...

Reply:
What real-world implication does increasing the cardinality have?

Reply:
The main issue (real-world implication) is that increasing cardinality of metrics will lead to overloading Prometheus. Thus, by not adding only what's minimally needed for us to debug problems at scale we make the SPO a noisy neighbour in terms of metrics